hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -130,6 +131,23 @@
 @SuppressWarnings({"checkstyle:MethodLength", "checkstyle:NPathComplexity"})
 public class TcpClientConnectionManager implements ClientConnectionManager, MembershipListener {
 
+    /**
+     * A private property to let users control the reconnection behavior of the client.
+     * <p>
+     * When enabled, the client will skip trying to connect the member list during
+     * reconnection attempts.
+     * <p>
+     * This property might be handy for users who are using the client with unisocket
+     * mode and exposing their multi-member cluster via a single load balancer or node port.
+     * In that scenario, the client would normally try to connect to the members from
+     * the member list first, but that would fail for sure, as the cluster members are not
+     * accessible from the client directly. In such use cases, setting this to
+     * {@code true} might make the reconnections shorter, as the client would directly
+     * try to connect to the configured load balancer/node port address from the configuration.
+     */
+    public static final HazelcastProperty SKIP_MEMBER_LIST_DURING_RECONNECTION =
```

Comment:
```suggestion
    public static final HazelcastProperty CLUSTER_CONNECTION_NO_MEMBER_LIST =
```

Reply:
I can change the name if you want, but I think the current version is simpler to understand